### PR TITLE
Preferences file read after deletion

### LINKED ISSUE #1626: Error about preferences broken blocking user from launching client
**OS**
Linux

**Wanna have the bug fixed quickly?**
Visit Issue hunt...
[Issue hunt](https://issuehunt.io/r/FAForever/downlords-faf-client)

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #1626

### CODE REVIEW

File: src/main/java/com/faforever/client/preferences/PreferencesService.java [hunk 1]
```diff
@@ -181,11 +182,16 @@ public static void configureLogging() {
...
+   * @return true if the file was deleted
    */
-  private void deleteFileIfEmpty() throws IOException {
+  private boolean deleteFileIfEmpty() throws IOException {
     if (Files.size(preferencesFilePath) == 0) {
       Files.delete(preferencesFilePath);
+      preferences = new Preferences();
+      return true;
     }
+    return false;
```
Comment (Geosearchef) [inline src/main/java/com/faforever/client/preferences/PreferencesService.java:194]: `else {
    return false;
}`

**Review by Geosearchef — APPROVED** [review]

File: src/main/java/com/faforever/client/preferences/PreferencesService.java (hunk 1 above)
Comment (1-alex98) [reply src/main/java/com/faforever/client/preferences/PreferencesService.java:194]: no why???

Comment (Geosearchef) [reply src/main/java/com/faforever/client/preferences/PreferencesService.java:194]: cause it's ten times more readable. No need to use the "abort" case here.

Comment (1-alex98) [reply src/main/java/com/faforever/client/preferences/PreferencesService.java:194]: we do this code style all over the application

**Pull request merged**